common/planners/src/plan_builder_test.rs

```diff
@@ -2,17 +2,20 @@
 //
 // SPDX-License-Identifier: Apache-2.0.
 
+use common_exception::Result;
+
+use crate::test::Test;
+use crate::*;
+
 #[test]
 fn test_plan_builds() -> anyhow::Result<()> {
-    use common_exception::Result;
     use pretty_assertions::assert_eq;
 
-    use crate::*;
-
     struct TestCase {
         name: &'static str,
         plan: Result<PlanNode>,
-        expect: &'static str
+        expect: &'static str,
+        err: &'static str
```

Comment:
It is neat to add the expected error here.

Reply:
err is expected error msg, empty str for normal return.